docs/devguide/debugging_guide.rst

```diff
@@ -36,44 +36,70 @@ Default discovery workflow
     title Default Discovery Workflow
     Server->RackHD: DHCP from PXE(nic or BIOS)
     RackHD->Server: ISC DHCP response with IP
+    note over RackHD:
+        If the node is already "known",
```

Comment:
Known is confusing, we should use discovered.

Reply:
@pengz1 A new node can be created manually with the API `POST /api/2.0/nodes`. So I think the original expression `known` is more suitable than `discovered` which only refers to the nodes that are discovered but not manually created.